dawn/src/dawn/AST/ASTStringifier.cpp

```diff
@@ -56,6 +56,11 @@ class StringVisitor : public ASTVisitor {
     scopeDepth_--;
   }
 
+  void visit(const std::shared_ptr<LoopStmt>& stmt) override {
+    ss_ << "for (" << stmt->getIterationDescr().toString() << ")\n";
+    stmt->getBlockStmt()->accept(*this);
```

Comment:
do we want to increase scope depth here also?

Reply:
the generated code is a complete mess anyways currently, and we rely on clang-format to make it readable. So far we weren't able to figure out how to do this in cmake (running clang-format on all generated files that is) though, so added for consistency's sake.

Reply:
I am ok with that